lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -481,7 +481,10 @@ bool oclIsBuiltin(StringRef Name, StringRef &DemangledName, bool IsCpp) {
   } else {
     size_t Start = Name.find_first_not_of("0123456789", 2);
     size_t Len = 0;
-    Name.substr(2, Start - 2).getAsInteger(10, Len);
+    if (Name.substr(2, Start - 2).getAsInteger(10, Len)) {
+      SPIRVDBG(errs() << "Error in extracting integer value");
+      return false;
+    }
     DemangledName = Name.substr(Start, Len);
```

Comment:
This is a micro-optimization that avoids an extra jump in scenarios where the condition is always true.
```suggestion
    if (Name.substr(2, Start - 2).getAsInteger(10, Len)) {
        DemangledName = Name.substr(Start, Len);
        return true;

    }
    SPIRVDBG(errs() << "Error in extracting integer value");
    return false;
```